x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/prepackaged_rules/windows_cve_2020_0601.json

```diff
@@ -1,36 +1,35 @@
-{
-  "description": "A spoofing vulnerability exists in the way Windows CryptoAPI (Crypt32.dll) validates Elliptic Curve Cryptography (ECC) certificates. An attacker could exploit the vulnerability by using a spoofed code-signing certificate to sign a malicious executable, making it appear the file was from a trusted, legitimate source.",
-  "index": [
-    "winlogbeat-*"
-  ],
-  "language": "kuery",
-  "max_signals": 33,
```

Comment:
Shouldn't this result in a version bump?

Reply:
we will re-check those in the next build.